spec/features/equipment_model_views_spec.rb

```diff
@@ -12,6 +12,7 @@
     shared_examples 'can view full and detailed table' do
       it_behaves_like 'can view detailed table'
       it { expect(page).to have_link 'Edit' }
+      it { should_not have_selector 'th', text: 'Ordering' }
```

Comment:
should use the `is_expected.not_to have_selector` syntax instead of `should_not`